### PR TITLE
Snippets for Sept per bug 1392702

### PR DESCRIPTION
**Pull request by peiying2**
I will update _b.lang once version _a.lang is approved.

### CODE REVIEW

File: en-US/snippets/2017/sep2017_a.lang [hunk 1]
```diff
@@ -0,0 +1,122 @@
+## NOTE: Reference bug 1392702. Deadline: 4 Sept, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# Snippet for Firefox Screenshots
+# Link: https://screenshots.firefox.com/#hello
+# "Complex screenshot hack" refers to traditional and more tedious method of taking and sharing screenshots.
+;Say goodbye to you complex screenshot hack. Screenshots is a new way to save and share within Firefox. <a>Learn more.</a>
+Say goodbye to you complex screenshot hack. Screenshots is a new way to save and share within Firefox. <a>Learn more.</a>
```
Comment (flodolo) [inline en-US/snippets/2017/sep2017_a.lang]: typo: your…

Also add a note that the "Screenshots" should not be localized.

File: en-US/snippets/2017/sep2017_a.lang [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+
+# Snippet for Firefox Screenshots
+# Link: https://screenshots.firefox.com/#hello
+# "Cool" in this context means the quality of being fashionably attractive or impressive. This message intends to convey that Firefox Screenshots is an easy way to save and share interesting images online.
+;Screenshots is a better way to screenshot. Save and share cool stuff without leaving Firefox. <a>Learn More.</a>
+Screenshots is a better way to screenshot. Save and share cool stuff without leaving Firefox. <a>Learn More.</a>
```
Comment (flodolo) [inline en-US/snippets/2017/sep2017_a.lang]: Same note here for Screenshots

File: en-US/snippets/2017/sep2017_a.lang [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+
+# Snippet for Firefox 55
+# Link:https://support.mozilla.org/kb/performance-settings
+# This message highlights that users are in control and they have the ability to customize Firefox.
+;1 processor, 4 processors, 7 processors? In Firefox 55, <a>the choice is yours.</a>
+1 processor, 4 processors, 7 processors? In Firefox 55, <a>the choice is yours.</a>
```
Comment (flodolo) [inline en-US/snippets/2017/sep2017_a.lang]: (unrelated to this PR) I'm not sure if this wanted to be funny (process, processor), but it turns out to be misleading: you can set the number of content processes, number of processors is something else :-\

File: en-US/snippets/2017/sep2017_a.lang [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+# Snippet for profiling age test
+# Link: https://addons.mozilla.org/firefox/
+# Power up" means to make better, to improve. The customization with add-ons will make the browser better.
+;Power up <a>Firefox with Add-ons.</a> Choose from dozens of customizations for privacy, social, gaming, and a whole lot more.
+Power up <a>Firefox with Add-ons.</a> Choose from dozens of customizations for privacy, social, gaming, and a whole lot more.
+
+
+# Snippet for profiling age test
+# Linke: https://www.mozilla.org/internet-health/
```
Comment (flodolo) [inline en-US/snippets/2017/sep2017_a.lang]: Link

Side note: id doesn't have that page translated…

File: en-US/snippets/2017/sep2017_a.lang [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+
+# Snippet for Firefox Accounts
+# Link: https://www.mozilla.org/firefox/accounts/
+# This message is for a user who doesn't have a Firefox account yet, and highlights the convenience of having an account will bring.
+;<a>Browse boldly.</a> Sign up for a Firefox Account to access your protected passwords, open tabs and bookmarks on all your devices.
+<a>Browse boldly.</a> Sign up for a Firefox Account to access your protected passwords, open tabs and bookmarks on all your devices.
+
+
+# Snippet for Firefox Accounts
```
Comment (flodolo) [inline en-US/snippets/2017/sep2017_a.lang]: It took me a while to figure out the different between these two snippets. Maybe explain in the note that one is about creating a new account, the other signing in an existing account?

File: en-US/snippets/2017/sep2017_a.lang (hunk 3 above)
Comment (peiying2) [inline en-US/snippets/2017/sep2017_a.lang]: Checked with Wes, he says to cut it.  Fixing others.

File: en-US/snippets/2017/sep2017_a.lang [hunk 6]
```diff
@@ -0,0 +1,124 @@
+## NOTE: Reference bug 1392702. Deadline: 4 Sept, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# Snippet for Firefox Screenshots
+# Screenshots remains in English
```
Comment (flodolo) [inline en-US/snippets/2017/sep2017_a.lang]: Better: "Screenshots (uppercase) is the name of the feature and has to be kept in English" (same later)

File: en-US/snippets/2017/sep2017_a.lang (hunk 5 above)
Comment (peiying2) [inline en-US/snippets/2017/sep2017_a.lang]: Shorten the comments to differentiate the two.

File: en-US/snippets/2017/sep2017_a.lang (hunk 6 above)
Comment (peiying2) [inline en-US/snippets/2017/sep2017_a.lang]: "has to be kept" -> "must be kept".  Does it sound too harsh?

Comment (flodolo) [inline en-US/snippets/2017/sep2017_a.lang]: I'm OK with both, as long as it's not "should be kept" ;-)

**Comment by flodolo** [comment]
Can you add files for other languages? Then we should be good to merge both.

**Pull request merged**